reservation_units/models.py

```diff
@@ -657,6 +658,20 @@ def state(self) -> ReservationUnitState:
 
         return ReservationUnitStateHelper.get_state(self)
 
+    def save(
+        self,
+        force_insert=False,
+        force_update=False,
+        using=None,
+        update_fields=None,
+        run_product_refresh=True,
```

Comment:
This is a bit of a hack to prevent infinite loop. `refresh_reservation_unit_product_mapping` task calls `save()` when product mapping is updated so I have to have a way to make sure the task is not triggered again.

Reply:
Also you could use queryset.update() => I'll add suggestion in proper place